### ISSUE TITLE
ipyvolume output is not at full resolution on HiDPI displays

### ISSUE THREAD

**Issue by astrofrog** [glue-viz/glue-jupyter#145]
When using glue-jupyter in Jupyter Lab, it looks like the ipyvolume widgets are pixellated when viewed on a HiDPI display:

![Screenshot from 2020-02-10 14-28-53](https://user-images.githubusercontent.com/314716/74158724-7b4d3080-4c12-11ea-92fe-2ffdf9cc246e.png)

(this can be seen compared to the buttons in the toolbar which are at full resolution)

**Comment by maartenbreddels** [comment]
Yes, this should be fixed in the next release of ipyvolume. I think I can push this out soonish. It does affect the performance (e.g. 4x more pixels). Do we want to control this?
